brave/brave-core
# Fixes Reset Profile dialog.

Fixes brave/brave-browser#2708

Instead of patching the html/js of the reset_profile_dialog, introduced
Brave behavior that hides/unchecks/disables the checkbox we don't want
seen along with the entire footer of the dialog since it contains
nothing else.

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:

1. Start Brave browser and navigate to brave://settings
2. Scroll down and click 'Advanced', scroll down again and click 'Restore settings to their original defaults'.
3. In the 'Reset settings' dialog, click 'Reset settings' button.
4. Verify that the dialog closes.


## Reviewer Checklist:
